### DISCUSSION TITLE
Make Treatment Suggestions Heuristic

### DISCUSSION THREAD

**Discussion by doc77** [NightscoutFoundation/xDrip#3010] [Ideas]
If not too difficult to program could an option for machine learning of C/I and correction treatments be implemented? 
Ideally a user entered starting point would be entered and then the app would make corrections to the treatment recommendations as more and more data points are entered.

Of course there are times when this model would not be good to use (ie sick days, etc). For this it would be valuable to be able to toggle on/off heuristics.

**Comment by Navid200** [comment]
I'm going to convert this to a discussion.  
The conversation can continue there.
